src/server/page.ts

```diff
@@ -147,14 +148,22 @@ export const getPage = (
     ctx.status = route.status
   }
 
-  const ADRTRACTION_NO = 'https://cdn.adt387.com/jsTag?ap=1492109567'
+  const ADTRACTION_NO = 'https://cdn.adt387.com/jsTag?ap=1492109567'
   const ADTRACTION_SE = 'https://cdn.adt387.com/jsTag?ap=1412531808'
+  const ADTRACTION_DK = 'https://cdn.adt387.com/jsTag?ap=1589794294'
+  const adtractionTag = match([
+    ['se', ADTRACTION_SE],
+    ['se-en', ADTRACTION_SE],
+    ['no', ADTRACTION_NO],
+    ['no-en', ADTRACTION_NO],
+    ['dk', ADTRACTION_DK],
+    ['dk-en', ADTRACTION_DK],
+  ])(ctx.params.locale)!
```

Comment:
Couldn't this just be a plain object? Then we could also easily type the `ctx.params.locale`s with the `LocaleLabel` type which is exported from `/client/l10n/locales.ts`? Maybe 😃

Reply:
Like 
```typescript
const adtractionTag: Record<LocaleLabel, string> = {
  'se': ADTRACTION_SE,
  'se-en': ADTRACTION_SE,
  'no': ADTRACTION_NO,
  'no-en': ADTRACTION_NO,
  'dk': ADTRACTION_DK,
  'dk-en': ADTRACTION_DK
```
or something like that 😀

Reply:
Or these values (the url's) could even be added to the `locales` object?

Reply:
Sure, we can do that! My only concern is that the `locales` object grows to contain a lot of different things, but I guess we can split it up later if we feel that way.

Reply:
Yeah my idea was to split it up in like on file for each locale object or something if they become to big and just import them and put them together in `localse.ts`. To me it makes sense to collect _all_ locale specific data in one place but you already know that since it was me who started building that structure haha 😄

Reply:
Sounds good! 🙌 Or have slice it on specific to features like ssn.